charts/pomerium/templates/proxy-deployment.yaml

```diff
@@ -79,6 +79,11 @@ spec:
             secretKeyRef:
               name: {{ $secretName }}
               key: shared-secret
+        - name: SIGNING_KEY
+          valueFrom:
+            secretKeyRef:
+              name: {{ template "pomerium.proxy.signingKeySecret.name" . }}
+              key: signing-key
```

Comment:
@tgomas I don't see the code that generates the secret anymore

Reply:
it's in `signing-key-secret.yaml`

Reply:
this seems to have been fixed